src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -2785,5 +2837,1166 @@ private void AssertValid()
+        //
+
+        /// <inheritdoc cref="IDivisionOperators{TSelf, TOther, TResult}.op_CheckedDivision(TSelf, TOther)" />
+        static BigInteger IDivisionOperators<BigInteger, BigInteger, BigInteger>.operator checked /(BigInteger left, BigInteger right) => left / right;
+
+        //
+        // IIncrementOperators
+        //
+
+        /// <inheritdoc cref="IIncrementOperators{TSelf}.op_CheckedIncrement(TSelf)" />
+        static BigInteger IIncrementOperators<BigInteger>.operator checked ++(BigInteger value) => ++value;
+
+        //
+        // IMultiplicativeIdentity
+        //
+
+        /// <inheritdoc cref="IMultiplicativeIdentity{TSelf, TResult}.MultiplicativeIdentity" />
+        static BigInteger IMultiplicativeIdentity<BigInteger, BigInteger>.MultiplicativeIdentity => One;
+
+        //
+        // IMultiplyOperators
+        //
+
+        /// <inheritdoc cref="IMultiplyOperators{TSelf, TOther, TResult}.op_CheckedMultiply(TSelf, TOther)" />
+        static BigInteger IMultiplyOperators<BigInteger, BigInteger, BigInteger>.operator checked *(BigInteger left, BigInteger right) => left * right;
+
+        //
+        // INumber
+        //
+
+        /// <inheritdoc cref="INumber{TSelf}.Clamp(TSelf, TSelf, TSelf)" />
+        public static BigInteger Clamp(BigInteger value, BigInteger min, BigInteger max)
+        {
+            value.AssertValid();
+            min.AssertValid();
+            max.AssertValid();
+
+            if (min > max)
+            {
+                ThrowMinMaxException(min, max);
+            }
+
+            if (value < min)
+            {
+                return min;
+            }
+            else if (value > max)
+            {
+                return max;
+            }
+
+            return value;
+
+            [DoesNotReturn]
+            static void ThrowMinMaxException<T>(T min, T max)
+            {
+                throw new ArgumentException(SR.Format(SR.Argument_MinMaxValue, min, max));
+            }
+        }
+
+        /// <inheritdoc cref="INumber{TSelf}.CopySign(TSelf, TSelf)" />
+        public static BigInteger CopySign(BigInteger value, BigInteger sign)
+        {
+            value.AssertValid();
+            sign.AssertValid();
+
+            int currentSign = value._sign;
+
+            if (value._bits is null)
+            {
+                currentSign = (currentSign >= 0) ? 1 : -1;
+            }
+
+            int targetSign = sign._sign;
+
+            if (sign._bits is null)
+            {
+                targetSign = (targetSign >= 0) ? 1 : -1;
+            }
+
+            return (currentSign == targetSign) ? value : -value;
+        }
+
+        /// <inheritdoc cref="INumber{TSelf}.CreateChecked{TOther}(TOther)" />
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static BigInteger CreateChecked<TOther>(TOther value)
+            where TOther : INumber<TOther>
+        {
+            if (typeof(TOther) == typeof(byte))
```

Comment:
What if TOther is typeof(BitInteger)?

Reply:
I'm going to be updating all the `Create` APIs at once. `BigInteger` (this PR) and `Complex` (https://github.com/dotnet/runtime/pull/68612) are the last two before all the "core types" implement the generic math interfaces.

The `INumberConverter` API + `Create` pattern that's going to be reviewed on Tuesday should simplify this, some of the logic duplication, and more generally cover types outside the "primitive set".

However, I can explicitly cover the `TSelf` conversions with this PR, if you'd prefer I do that.

Reply:
Updated to cover the "identity" conversion for this PR.